benches/stress/codec.js

```diff
@@ -1,16 +1,18 @@
+'use strict';
+
 var json = {
 	encode: JSON.stringify,
 	decode: JSON.parse
 };
 
-var MsgPack = require('node-msgpack');
-msgpack = {
-	encode: MsgPack.pack,
-	decode: function(str) { return MsgPack.unpack(new Buffer(str)); }
-};
+// var MsgPack = require('node-msgpack');
+// var msgpack = {
+// 	encode: MsgPack.pack,
+// 	decode: function(str) { return MsgPack.unpack(new Buffer(str)); }
+// };
```

Comment:
`module.exports = msgpack;` is already commented out and therefor this is a unused variable. I was not sure what this part was for and normally I'd just delete unused code, but I'd prefer to know the reason for the original code before deleting it :).